doc/RobotFirmware2015-OSX.md

```diff
@@ -0,0 +1,86 @@
+# OSX Firmware Setup
```

Comment:
That's the plan, things are kinda messy at the moment though. Figured this was next best thing until a script gets in place.